pre-docs/bolt_new_features.md

```diff
@@ -0,0 +1,417 @@
+# New features
```

Comment:
this file should be deleted right?

Reply:
Yup. Was keeping it around for safety, but I've deleted it now in this branch.